src/app/mat-select-search/mat-select-search.component.ts

```diff
@@ -337,24 +337,24 @@ export class MatSelectSearchComponent implements OnInit, OnDestroy, ControlValue
             // avoid "expression has been changed" error
             setTimeout(() => {
               // Convert the QueryList to an array
-              const options = this._options.toArray();
+              const options = this._options.toArray() as MatOption[];
```

Comment:
nit: this typecast should not be necessary, since  
https://github.com/angular/angular/blob/ee5b1cf076cb977b86bbe265e5243f145f4d1e4a/packages/core/src/linker/query_list.ts#L136  already has the correct return type, no?

Reply:
yeah i removed it, i guess vscode didn't show it correctly the first time